### PR TITLE
[docs] batch prediction guide, strict mode

### PR DESCRIPTION
**Pull request by maxpumperla**

### CODE REVIEW

File: doc/source/data/batch_inference.rst [hunk 1]
```diff
@@ -498,6 +497,18 @@ namely Pandas, NumPy, and Arrow, and how they're used in Ray Data:
     :start-after: __simple_numpy_start__
     :end-before: __simple_numpy_end__
 
+.. tabbed:: Pandas
+
+  The ``"pandas"`` batch format presents batches in
+  `pandas.DataFrame <https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.html>`__
+  format. If converting a simple dataset to Pandas DataFrame batches, a single-column
+  dataframe with the column ``"__value__"`` will be created.
```
Comment (ericl) [inline doc/source/data/batch_inference.rst:505]: Remove all mention of `__value__` or "single-column" datasets.

**Review by amogkam — APPROVED** [review]
awesome, thanks a lot! lgtm once eric's comments are addressed

**Comment by amogkam** [comment]
@ericl your comments are addressed

File: doc/source/data/batch_inference.rst (hunk 1 above)
Comment (amogkam) [reply doc/source/data/batch_inference.rst]: done

File: doc/source/data/batch_inference.rst [hunk 2]
```diff
@@ -498,6 +497,18 @@ namely Pandas, NumPy, and Arrow, and how they're used in Ray Data:
     :start-after: __simple_numpy_start__
     :end-before: __simple_numpy_end__
 
+.. tabbed:: Pandas
```
[the remark this answers is not in the document]
Comment (amogkam) [reply doc/source/data/batch_inference.rst:488]: removed

**Review by ericl — APPROVED** [review]

**Pull request merged**